### PR TITLE
added objc anotation

### PR DESCRIPTION
**Pull request by Medtrack**
Hello, 
First, thanks for your great work!

I have a custom (not public) cordova plugin usign a the swift 4 version for iOS.
I was integrating your plugin in my app and realized about the @objc annotations. These are required on swift 4 for methods. Since my plugin is bigger and uses swift 4 I cannot change the version on my config.xml. 

I forked your plugin and added the anotiations for you to know. I don't think it will break anything to use the annotations and doing so your great plugin is compatible with Swift4.

### CODE REVIEW

**Comment by NiklasMerz** [comment]
Thank you for your feedback. If the CI build is successful and my it builds with the latest xcode version this should be good.

**Comment by Medtrack** [comment]
Sorry for the delay: thank you!

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 node_modules/
-package-lock.json
\ No newline at end of file
+package-lock.json
+.idea/
```
Comment (NiklasMerz) [inline .gitignore]: Please remove `.idea/`

**Comment by Medtrack** [comment]
Hi again,

I changed .gitignore file. 

Thanks :-)

**Pull request merged**